utils/generator/src/Command/Typo3GenerateCommand.php

```diff
@@ -42,8 +41,8 @@ public function __construct(
 
     protected function configure(): void
     {
-        $this->setName(CommandNaming::classToName(self::class));
-        $this->setAliases(['typo3-create', 'typo3-rector']);
+        $this->setName('typo3-generate');
+        $this->setAliases(['typo3-create']);
         $this->setDescription('[DEV] Create a new TYPO3 Rector, in a proper location, with new tests');
```

Comment:
I've checked the docs and "typo3-rector" is always used for the name of this project.
So I think the "create" or "generate" would be better name.